unitex/src/fr/umlv/unitex/frames/ConcordanceFrame.java

```diff
@@ -61,10 +54,11 @@
 
 /**
  * This class describes a frame that can show an HTML concordance file.
- * 
+ *
  * @author Sébastien Paumier
  */
 public class ConcordanceFrame extends TabbableInternalFrame {
+  private final File f;
```

Comment:
can we have a better name for the variable f? concordanceFile or concordanceFrameFile ?